tests/test_ses/test_ses_boto3.py

```diff
@@ -484,3 +485,66 @@ def test_create_ses_template():
 
     result = conn.list_templates()
     result["TemplatesMetadata"][0]["Name"].should.equal("MyTemplate")
+
+
+@mock_ses
+def test_render_template():
+    conn = boto3.client("ses", region_name="us-east-1")
+
+    kwargs = dict(
+        TemplateName="MyTestTemplate",
+        TemplateData=json.dumps({"name": "John", "favoriteanimal": "Lion"}),
+    )
+
+    conn.test_render_template.when.called_with(**kwargs).should.throw(ClientError)
```

Comment:
When testing error scenarios, we prefer to use pytest.raises, as that makes it easier to assert that the correct error was thrown:
```
with pytest.raises(ClientError) as ex:
  # something that fails
err  = ex.value.response["Error"]
err["Code"].should.equal("...")
err["Message"].should.equal("...")
```